.travis.yml

```diff
@@ -14,14 +14,6 @@ services:
 env:
   - WP_VERSION=latest WP_MULTISITE=0  WC_VERSION=3.4.5
 
-# Additonal tests against stable PHP (min recommended version is 5.6) and past supported versions of WP.
-matrix:
-  include:
-  - php: 5.6
-    env: WP_VERSION=latest WP_MULTISITE=1 WC_VERSION=2.6.14
-  - php: 7.1
-    env: WP_VERSION=latest WP_MULTISITE=0 WC_VERSION=3.4.5
```

Comment:
The only test that was removed was the one with `WC_VERSION=2.6.14` since we don't support it anymore. Do you think it would be better to create a new issue to add new test configurations, or should we do it in this PR?

Reply:
I agree this is of out of the scope of this PR. Let's create a new issue and fix it later.

Reply:
Created https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1199.